### PR TITLE
docs(anexia): add info for templates

### PR DESCRIPTION
**Pull request by ahmedwaleedmalik**
Signed-off-by: Waleed Malik <ahmedwaleedmalik@gmail.com>

**What this PR does / why we need it**:
Templates are rotated quite often and the old templates are removed as a result.This PR intends to document the process of retrieving a list of available templates.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Optional Release Note**:

```release-note
NONE
```

### CODE REVIEW

File: docs/anexia.md [hunk 1]
```diff
@@ -6,4 +6,14 @@ This provider implementation is currently in **alpha** state.
...
-An example machine deployment can be found here: [examples/anexia-machinedeployment.yaml](../examples/anexia-machinedeployment.yaml)
\ No newline at end of file
+An example machine deployment can be found here: [examples/anexia-machinedeployment.yaml](../examples/anexia-machinedeployment.yaml)
+
+## Templates
+
+To retrieve all available templates against a given location:
+
+```
+https://engine.anexia-it.com/api/vsphere/v1/provisioning/templates.json/<location identifier>/templates?page=1&limit=50
```
Comment (embik) [inline docs/anexia.md:16]: Just curious, does this work without any credentials? I don't know any location identifiers so no idea how to try it out. Maybe we can supplement a CLI call that prints available templates?

Comment (ahmedwaleedmalik) [reply docs/anexia.md:16]: You need the authentication token for this https://engine.anexia-it.com/api/vsphere/doc/

Comment (ahmedwaleedmalik) [reply docs/anexia.md:16]: This is not supposed to be an unauthenticated/public endpoint.

Comment (embik) [reply docs/anexia.md:16]: Ah, gotcha. Can we include that information in the doc maybe?

Comment (ahmedwaleedmalik) [reply docs/anexia.md:16]: I don't think it's required but sure :)

**Review by embik — APPROVED** [review]

**Comment by ahmedwaleedmalik** [comment]
/retest 

Failures due to webhook timeouts 🤷‍♂️

**Pull request merged**